packages/vscode/package.json

```diff
@@ -49,10 +49,6 @@
       }
     ],
     "commands": [
-      {
-        "command": "astro.reloadProjects",
-        "title": "Astro: Reload Projects"
-      },
```

Comment:
Does this command not exist anymore in 2.2? People use it fairly often to workaround caching issues

Reply:
Yes it was removed in the refactor, I didn't expect it to be used, I will add it back.

Reply:
Added by [`18a302a` (#863)](https://github.com/withastro/language-tools/pull/863/commits/18a302adb862a400903e7329ddbb7ae7ea75c078)!